pylint-2.7

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env vpython3
```

Comment:
Also, could you please document why these files are needed, and the off-the-shelf pylint that had been working doesn't work anymore without these?

Reply:
I made a copy of the pylint in this repo in order to add the `requests` dependency so that it would be available to vpython in the linting stage, I'll add that documentation if this approach is ok

Reply:
Why do we need the requests dependency? It looks like it's only being used in once place to do an http request?